client/components/contact/styles/contactimage.scss

```diff
@@ -0,0 +1,20 @@
+
+.contact-image-cover {
```

Comment:
For now, I think it would be best to keep the codebase consistent for now and use `makeStyles` until we come to a decision about `makeStyles` vs. scss files. Wdyt?

Reply:
updated in the latest push